dynawo/sources/Models/CPP/Controls/Voltage/ModelMinMaxMean/DYNModelMinMaxMean.cpp

```diff
@@ -0,0 +1,365 @@
+//
+// Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+// simulation tool for power systems.
+//
+
+/**
+ * @file  DYNModelMinMaxMean.cpp
+ *
+ * @brief Reference frequency model implementation
+ *
+ * All generators of the network are connected to this model
```

Comment:
same

Reply:
All these comments (in the PR and in the files!) should be sorted now :)